ArduPlane/mode_rtl.cpp

```diff
@@ -8,10 +8,12 @@ bool ModeRTL::_enter()
     plane.rtl.done_climb = false;
 #if HAL_QUADPLANE_ENABLED
     plane.vtol_approach_s.approach_stage = Plane::Landing_ApproachStage::RTL;
-#endif
+
+    if (plane.quadplane.guided_wait_takeoff_on_mode_enter) {
+       plane.set_mode(plane.mode_qland, ModeReason::QLAND_INSTEAD_OF_RTL);
```

Comment:
```suggestion
       plane.set_mode(plane.mode_qland, ModeReason::QLAND_INSTEAD_OF_RTL);
       return true;
```
to stop switching to QRTL in the `switch_QRTL` check

Reply:
thanks, done